### ISSUE TITLE
train

### ISSUE THREAD

**Issue by 13sunmin** [saikatdutta/Stacked_DMSHN_bokeh#7]
Hello Sir,

While trying to replicate your work, I'm having some difficulties in training the model. The Adam optimizer is returning nan, and am not sure if its due to the wrong implementation of the loss function (tried implementing after reading your paper). It would be greatly helpful if you could share your training script. You could send it to my email id : 849972256@qq.com

Thank you.

**Comment by saikatdutta** [comment]
Hi, 

Thanks for showing interest in our work. We've lost the training script, so I won't be able to share the same. However, I can advise you the following:
1. Make sure you are using the correct pytorch version. 
2. We have linked the loss function repositories in "Useful Repositories" (check Readme). 
3. We used an older version of MS-SSIM. Please undo this change: https://github.com/jorge-pessoa/pytorch-msssim/commit/ff882263779e94c29ec4e76a432d2ec80df8796b#diff-b583e4c6807ec522fa88e389e2245135a1686c73bbcdfa8baa8a7068288cf49f
4. You can start by training with a simple loss function (L1) to check if there is some other problem.

Let me know if these fixes work.

**Comment by saikatdutta** [comment]
Closing. Feel free to open if needed.
